packages/signals/browser-signals/src/types/signals.ts

```diff
@@ -45,10 +49,28 @@ export type InstrumentationSignal = AppSignal<
   InstrumentationData
 >
 
+type NetworkRequestData = {
+  action: 'Request'
+  url: RequestInfo | URL
+  method: string
+  data: {}
+}
+
+type NetworkResponseData = {
+  action: 'Response'
+  url: string
+  data: {}
```

Comment:
This type is an empty object, the best type would be Record<string, unknown> or {[key: string]: unknown}